firmware_DavidBoard/Core/Inc/TemperatureCalc.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+/**
+ * This module turns ADC voltage readings into temperature (in Celsius)
+ * Author(s): Jacob Smith
+ */
+
+#include "Common.h" //move these to TemperatureCalc.c
```

Comment:
Common.h can stay. It's the only header present everywhere. But yes, I2CManager needs to go to your .c file.